posts/2023-12-12-23.0.0.12.adoc

```diff
@@ -0,0 +1,533 @@
+----
+
+For more information, see xref:/docs/latest/reference/feature/openidConnectClient-1.0.html#access-token[Check the access token for user and group information].
+
+[#CVEs]
+== Security vulnerability (CVE) fixes in this release
+[cols="5*"]
+|===
+|CVE |CVSS Score |Vulnerability Assessment |Versions Affected |Notes
+
+|http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-44487[CVE-2023-44487]
+|7.5
+|Denial of service
+|18.0.0.2 - 23.0.0.11
+|Affects the link:{url-prefix}/docs/latest/reference/feature/servlet-3.1.html[servlet-3.0], link:{url-prefix}/docs/latest/reference/feature/servlet-4.0.html[servlet-4.0], link:{url-prefix}/docs/latest/reference/feature/servlet-5.0.html[servlet-5.0] and link:{url-prefix}/docs/latest/reference/feature/servlet-6.0.html[servlet-6.0] features
+|===
+
+For a list of past security vulnerability fixes, reference the link:{url-prefix}/docs/latest/security-vulnerabilities.html[Security vulnerability (CVE) list].
+
+// ------ <Excerpt From Previous Post: End> ------
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // //
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/27087
+// Contact/Reviewer: yeekangc
+// // // // // // // //
+[#maven]
+== Liberty Maven plug-in 3.10 and Liberty Gradle plug-in 3.8
+
+New releases for Liberty Maven and Gradle plug-ins are now available. The following notable new features are included:
+
+* Support for deploying Spring Boot 3 applications to Liberty by using the build plug-ins through Liberty's `springBoot-3.0` feature
+* Support for running the plug-ins with Java 21
+
+
+To use the new Maven plug-in version, specify the 3.10 release in your Maven `pom.xml` file.
+For Gradle, specify the 3.8 release in your `build.gradle` file.
+
+For more information about these plug-ins, see the following resources:
+
+* link:https://github.com/OpenLiberty/ci.maven/releases[Liberty Maven plug-in 3.10 release notes]
+* link:https://github.com/OpenLiberty/ci.gradle/releases[Liberty Gradle plug-in 3.8 release notes]
+
+For more information about Spring Boot support with the Liberty Maven plug-in, see link:https://github.com/OpenLiberty/ci.maven/blob/main/docs/spring-boot-support.md[ci.maven: Spring Boot Support].
+For more information about Spring Boot support with the Liberty Gradle plug-in, see link:https://github.com/OpenLiberty/ci.gradle/blob/main/docs/spring-boot-support.md[ci.gradle: Spring Boot Support].
+
+
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC>
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // //
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/27086
+// Contact/Reviewer: yeekangc
+// // // // // // // //
+
+[#lt]
+== Liberty Tools 23.0.12 for Eclipse IDE, IntelliJ IDEA, and Visual Studio Code
+
+Liberty Tools now support the latest available releases of Eclipse IDE, IntelliJ IDEA, and Visual Studio Code. This release also includes various enhancements and fixes.
```

Comment:
> Liberty Tools now support the latest available releases of Eclipse IDE,

This is a bit of a tricky statement .

Presumably we want some reusable text here that we can copy/paste with each release. 

The problem with saying "support the latest" is it won't always be true.  The latest release is a moving target.. e.g. a new one was released in the middle of the process of producing this doc on Dec. 6th.   Our general target is the N-1 previous release.     

Can we "soften" the claim a bit, like:    
   "Liberty Tools has been updated to support newer releases of Eclipse IDE, .... "
      OR
   "Liberty Tools now supports newer available releases of Eclipse IDE, .... "

Reply:
@scottkurz - edited to

> Liberty Tools now support newer releases of Eclipse IDE, IntelliJ IDEA, and Visual Studio Code.